docs/developer.md

```diff
@@ -60,11 +60,29 @@ yarn run build
 
 Once completed, the Lookout UI should be accessible through your browser at `http://localhost:8089`
 
-For UI development you can also use the React development server. Note that the Lookout API will still have to be running for this to work.
+For UI development, you can also use the React development server and skip the build step. Note that the Lookout API service will 
+still have to be running for this to work. Browse to `http://localhost:3000` with this.
 ```bash
 yarn run start
 ```
 
+## Debugging localdev
+
+With the `localdev` environment running, it's possible to replace one more more services with 
+a debugging instance. Below are the steps for replacing the `server` container with a debugging session.
+
+```bash
+cd localdev
+docker stop server && docker rm server
+docker-compose run --entrypoint bash --name server server
+
+root@11c57f54c993:/app# go install github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
👏🏼 This is great to give contributors precise steps on getting Delve running here.